CHANGES.md

```diff
@@ -11975,7 +11975,7 @@ OpenSSL 0.9.7.]
    *Steve Henson*
 
  * Support for crypto accelerator cards from Accelerated Encryption
-   Processing, www.aep.ie.  (Use engine 'aep')
+   Processing, www.aep.i.e.  (Use engine 'aep')
```

Comment:
I suspect that `www.aep.ie.` -> `www.aep.i.e.` glitch crept in by automatic substitution.

BTW, shouldn't `i.e.` and `e.g.` always be surrounded by a commas, e.g., `, i.e.,`?
Not that I would request such a change in all the docs, but I'm asking just for being able to do the right thing in new texts.

Reply:
This is fixed in this branch since 2021-10-26 10:46:54.

In the American style of writing, a comma is inserted before and after "i.e."; in the British style of writing, a comma is inserted before but not after "i.e."... so, since the documentation and source code follows the British style... to work on this would be another round of changes.